Dockerfile

```diff
@@ -5,6 +5,10 @@ ENV DOCKER_HOME /usr/src/app
 
 RUN apk update && apk upgrade
 
+# Update the package list and upgrade libpq and postgresql15-client
+RUN apk update && apk upgrade && \
+    apk add --no-cache libpq postgresql15-client 
+
 # Temporary solution to dump and restore database
```

Comment:
This was a temporary solution to dump data as it mentions, then why did we need the above line 10?

Reply:
@hemantgoswami , Line 9 & 10 are added newly to fix vulnerability on libpq & postgresql15.
Line 12 is for below.
# Temporary solution to dump and restore database
RUN apk add --no-cache --update postgresql-client